torchci/pages/tts/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -130,13 +129,13 @@ function Graphs({
     return <Skeleton variant={"rectangular"} height={"100%"} />;
   }
 
-  let startTime = queryParams.find((p) => p.name === "startTime")?.value;
-  let stopTime = queryParams.find((p) => p.name === "stopTime")?.value;
+  let startTime = (queryParams as any)["startTime"];
```

Comment:
IIRC, if you declare `queryParams` as `{ [k: string]: any }`, then casting is not needed here